managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+*Embedded Workflows* are an alternative to the *Embedded Automate* capability for authoring and running automation. With *Embedded Workflows*, you can continue to use the *Embedded Automate* feature for service catalog creation and management.
+
+### Enabling Embedded Workflows
+
+The Embedded Workflows functionality is not enabled and available by default. A user with administrator rights must enable the functionality by turning on a prototype flag. If you are an administrative user, you can complete the following steps to enable the functionality:
+
+1. From the {{site.data.product.title_short }} UI, click **Settings** > **Applications Settings**. Then, click the **Advanced** tab.
+2. On the *Advanced* tab, search for `ems_workflows` within the *Prototype* section. For the entry, set the `enabled` setting to `true`.
+3. Click **Save**.
+4. Restart the application.
+
+When the application is restarted, the *Embedded Workflows* functionality is enabled. <!-- This functionality reuses existing API. -->
+
+### Embedded Workflows UI additions
+
+When the *Embedded Workflows* functionality is enabled, new **Automation** > **Embedded Workflows** pages become available for use in the UI. In addition to the *Embedded Workflows* UI, you can now select **Embedded Workflows** as a dropdown option along with **Embedded Automate** for various fields on other UI pages.
+
+#### Embedded Workflow UI
+
+The new *Embedded Workflows* functionality adds the following pages to the UI, which you can use to access and work with workflows. To view these pages, go to **Automation** > **Embedded Workflows** and select the page.
+
+![Embedded Workflow UI](../images/embedworkflow_ui.png){: caption="Embedded Workflow UI" caption-side="bottom"}
+
+- **Workflows**
+
+   This UI page includes a summary of the available workflows. You can use different views to review the workflows:
+
+   - Text view, which is the raw representation (JSON or YAML) of the Amazon State Languages (ASL) code.
+   - Graph view, which shows an SVG visual representation of the workflow structure.
+
+- **Repositories**
+
+   This page lists, and provides a summary of, the added embedded workflow repositories. From this page, you can add, update, and remove repositories.
+   When a repository is added and synchronized, any workflows that are found in the repository can be viewed on the **Workflows** UI page.
+
+- **Credentials**
+
+   Credentials are used by {{ site.data.product.title_short }} for authentication when running embedded workflows against machines, synchronizing with inventory sources, and importing project content from a version control system.
+
+   This page lists, and provides a summary of, the credentials for accessing and using embedded workflows. From this page, you can add, update, and delete credentials.
+
+   These credentials are tied to the provider instance, and are used by the workflow when it runs. The credential types are defined by the provider.
+
+#### UI Field Updates
```

Comment:
I don't think we should say "UI Field Updates" because this reads more like a changelog (aka here is what we added) and less like long-term documentation.

Reply:
I have removed this header and updated the section.